corehq/apps/app_manager/models.py

```diff
@@ -3786,7 +3786,7 @@ def root_module_id(self):
     def get_suite_forms(self):
         if not self.source_module:
             return []
-        return self.source_module.get_forms()
+        return (f for f in self.source_module.get_forms() if f.unique_id not in self.excluded_form_ids)
```

Comment:
If there's not a specific reason for using a generator object, it's definitely avoids very unintuitive bugs for this function to return a list rather than a generator object:

```
suite_forms = self.get_suite_forms()
for form in suite_forms:
    # do stuff
# then later
for form in suite_forms:
    # try to do other stuff
    # this will never get reached because suite_forms is already used up
```